Modules/ExtendedPlayerControl.cs

```diff
@@ -249,17 +251,71 @@ public static void RpcBootFromVentDesync(this PlayerPhysics physics, int ventId,
         writer.WritePacked(ventId);
         AmongUsClient.Instance.FinishRpcImmediately(writer);
     }
+
+    public static void RpcRevive(this PlayerControl player, RoleTypes roleTypes, bool IsDesyncImpostor = false, List<PlayerControl> FellowImps = null)
+    {
+        if (player.Data.IsDead == false || roleTypes is RoleTypes.GuardianAngel or RoleTypes.CrewmateGhost or RoleTypes.ImpostorGhost)
+        {
+            Logger.Warn($"Invalid Revive for {player.GetRealName()} of roletype: {roleTypes} / Player was already alive? {!player.Data.IsDead}", "RpcRevive");
+            return;
+        }
+
+        player.RpcChangeRoleBasis(roleTypes, IsDesyncImpostor, FellowImps);
+        Main.PlayerStates[player.PlayerId].IsDead = false;
```

Comment:
Bruh again need RPC

Reply:
Maybe make Rpc for Sync some data in PlayerStates?

Reply:
IsDead
IsDissconected
DeathReason

and somthing else

Reply:
omg I forgor 💀

Reply:
IsDead
IsDissconected
DeathReason
playerstate
mainallplayercooldonw
mainallaplayerspeed

I think